### DISCUSSION TITLE
Questions about binding in Roslyn

### DISCUSSION THREAD

**Discussion by FilipToth** [dotnet/roslyn#54139] [Q&A] [answered]
Hi, all!

Is there a way to view resulting Bound trees (Maybe in CSI, or the Roslyn SDK)? Maybe a method that would return the bound tree in whatever class structure.
Thank you in advance.

**Comment by CyrusNajmabadi** [comment, marked as answer]
If you use the IOperation tree (semanticModel.GetOperation), you will effectively get the public facing side of our bound nodes.

**Reply by FilipToth** [reply]
Alright, that sounds good. Thank you!
